File: Application/Classes/Controls/COLabelLanguage/COLabelLanguage.m
//
//  COLabelLanguage.m
//  CoAssets
//
//  Created by Macintosh HD on 1/7/16.
//  Copyright © 2016 Sanyi. All rights reserved.
//

#import "COLabelLanguage.h"

@implementation COLabelLanguage

- (void)awakeFromNib {
    [super awakeFromNib];
    NSString *text = self.text;
    if (![[text trim] isEmpty]) {
        self.text = m_string(text);
    }
}

@end

File: CoAssetsApps/COCore/Classes/Controls/Others/COLabelLanguage.swift
//
//  COLabelLanguage.swift
//  CoAssetsApps
//
//  Created by Nikmesoft_M008 on 04/03/2016.
//  Copyright © 2016 TruongVO07. All rights reserved.
//

import UIKit

class COLabelLanguage: UILabel {

    //MARK: Override
    override func awakeFromNib() {
        super.awakeFromNib()
        if let text = self.text where text.trim().isEmpty {
            self.text = m_local_string(text)
        }  
    }
}
